tasks/glance_image.yml

```diff
@@ -1,5 +1,12 @@
 ---
 - block:
+  - name: Check if template is correct
+    fail:
+      msg: "'templates' variable does not contain mandatory parameters '{{item}}"
```

Comment:
This role actually don't have any `templates` variable, so this error  message should look like:

```
"mandatory parameter '{{ item }}' is not defined"
```